NOTICE.txt

```diff
@@ -0,0 +1,2 @@
+OpenSearch (https://opensearch.org/)
```

Comment:
After this will he have license headers for every file?

Reply:
https://github.com/opensearch-project/.github/pull/29

Reply:
Yes, all the new files needs to add these two lines.
```
New files in your code contributions should contain the following license header. If you are modifying existing files with license headers, or including new files that already have license headers, do not remove or modify them without guidance.
```

```
# Copyright OpenSearch Contributors
# SPDX-License-Identifier: Apache-2.0
```

Reply:
Ok so we will only add it in the new files that we create and keep the old ones as it is?

Reply:
I think it means all the old files WITH EXISTING HEADER can stay the same, means files without headers needs to get added with the new ones.